### DISCUSSION TITLE
Minimum PHP version issue

### DISCUSSION THREAD

**Discussion by snapey** [laravel/framework#33697] [General] [answered]
Who's responsibility is it to fix this?

Laravel requires mockery/mockery "^1.3.1" but version 1.4.1 is now available

mockery 1.4.1 requires at least php 7.3 which is conflict with the min stated requirements for Laravel

Should Laravel composer peg mockery at 1.3.1 or should the minimum php version be upgraded for the framework also?

Currently it is not possible to run an install on an environment with php 7.2.x without doctoring the composer file

**Comment by GrahamCampbell** [comment, marked as answer]
That's not a bug, because Composer on PHP 7.2 will just resolve the newest version of mockery that is still compatible, which will be Mocker 1.3.2. For those using the Laravel installer, you need PHP 7.3 or higher. PHP 7.2 users can just use composer's "create-project" command.

**Reply by GrahamCampbell** [reply]
NB We have tests that prove this works.
